SmithyDafnySedMakefile.mk

```diff
@@ -0,0 +1,82 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# These are make targets that use sed to replace extern names.
+# Some Smithy-Dafny modules declare extern attributes with names that do not work in all languages.
+# These targets can remove or add those extern names based on the target language.
+# A target language SHOULD declare use of these targets.
+
+# These should eventually be replaced by replaceable modules that declare per-langauge extern names.
+# Eventually, the replaceable modules should be removed (or, their extern names should be removed),
+# and should be replaced by per-language package/namespace prefixes.
```

Comment:
```suggestion
# and should be replaced by per-language package/namespace prefixes.
See https://github.com/smithy-lang/smithy-dafny/issues/528.
```

Reply:
I put this in a slightly different place for clarity on what is actually addressed by #528; I propose that #528 is just an alternative interim solution